src/hooks/bridge/useBridgeCallback.ts

```diff
@@ -292,7 +294,15 @@ function useBridgeSwap(
                 trade_qty: typedValue,
               },
             })
-            sendTxToKsSetting(chainId, chainIdOut, txHash, from_token, to_token, inputAmountStr, outputAmountStr ?? '')
+            sendTxToKsSetting(
+              chainId,
+              chainIdOut,
+              txHash,
+              from_token,
+              to_token,
+              inputAmountStr,
+              outputInfo?.outputAmount?.toString() ?? '',
```

Comment:
save db raw string

Reply:
Please use `outputAmountStr`.

Reply:
No need save raw string, save formatted string pls.

Reply:
Rounded to 6 digits is totally fine.